storio-common-annotations-processor/src/main/java/com/pushtorefresh/storio/common/annotations/processor/StorIOAnnotationsProcessor.java

```diff
@@ -98,10 +100,13 @@ private TypeElement validateAnnotatedClass(@NotNull final Element annotatedEleme
      * Checks that element annotated with {@link StorIOColumnMeta} satisfies all required conditions.
      *
      * @param annotatedElement an annotated field
+     * @return boolean flag which indicates if annotated element valid or not
      */
-    protected void validateAnnotatedFieldOrMethod(@NotNull final Element annotatedElement) {
+    protected boolean validateAnnotatedFieldOrMethod(@NotNull final Element annotatedElement) {
```

Comment:
I mean I can change to a checked exception, but I think it will be even worse than the current implementation. We can't recover from that state and we don't need to print anything for user, so it will be just an empty catch block.